lists/by.csv

```diff
@@ -295,3 +295,50 @@ https://tsepkalo.by/,POLR,Political Criticism,2020-07-28,ifreedombelarus,
 https://help.honestpeople.by/,POLR,Political Criticism,2020-07-28,ifreedombelarus,
+https://primaries.by/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://masheka.by/,SRCH,Search Engines,2020-08-24,OONI,Reportedly blocked
+https://progomel.by/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://tip.by/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://virtualbrest.by/,SRCH,Search Engines,2020-08-24,OONI,Reportedly blocked
+https://vkurier.by/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://015.by/search/,SRCH,Search Engines,2020-08-24,OONI,Reportedly blocked
+https://tsepkalo.info/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+http://tsepkalo.com/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://honestby.org/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://agitka2020.com/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://opg.ucoz.net/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+http://vnezakona.net/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://narodny-opros.info/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://statkevich.org/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://belarus.regnum.ru/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://eurobelarus.info/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+http://mfront.net/,PUBH,Public Health,2020-08-24,OONI,Reportedly blocked
+http://spring96.org/ru/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://ucpb.info/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+http://svaboda2.net/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://bchd.info/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://belarusinfocus.info/by/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://belprauda.org/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://elections2020.spring96.org/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://flagshtok.info/by/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://mspring.online/ru/tsikhanouskaja-on-top/,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://www.politnavigator.net/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://vitebskspring.org/,HUMR,Human Rights Issues,2020-08-24,OONI,Reportedly blocked
+https://vot-tak.tv/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+http://www.moyby.com/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://mediazona.by/chronicle/belvybary,POLR,Political Criticism,2020-08-24,OONI,Reportedly blocked
+https://by.tribuna.com/,NEWS,News Media,2020-08-24,OONI,Reportedly blocked
+https://cock.li/,COMT,Communication Tools,2020-08-24,OONI,Reportedly blocked
+https://protonmai1.com/,COMT,Communication Tools,2020-08-24,OONI,Reportedly blocked

```

Comment:
Is this mispelled? Was it supposed to be `protonmail.com`?

FYI, I did not see any HTTP or HTTPS interference for protonmail.com on Bel Telecom using the Censored Planet data. Though I know there were brief moments of internet shutdown, which could explain the reports. Or perhaps it was DNS.

Reply:
@fortuna Good catch! protonmai1.com was included in the list shared with me -- I'll delete it. Protonmail.com appears to be accessible indeed: https://explorer.ooni.org/search?until=2020-08-26&domain=protonmail.com&probe_cc=BY (most of the anomalies are due to an old bug in the python legacy version of ooniprobe that some people are still running) Though I'm being told that these URLs are only now being blocked...

Reply:
I see https://protonmai1.com on this list: https://translate.google.com/translate?sl=ru&tl=en&u=https%3A%2F%2F42.tut.by%2F697663

But I have the impression that was a misspelling. That address doesn't load. The real address should definitely be there.

On another note, I found out that Proton Mail wasn't blocked before or during the shutdown, but right after:
https://twitter.com/vinifortuna/status/1298763471031394304